src/server/db/migration/schema.ts

```diff
@@ -379,6 +379,37 @@ export const chats = createTable(
   }),
 )
 
+export const chat_feedback = createTable(
+  'chat_feedback',
+  {
+    id: uuid('id').notNull().primaryKey(),
+    chatId: uuid('chat_id')
+      .notNull()
+      .references(() => chats.id),
+    typeId: integer('feedback_type').notNull(),
+    notes: text('notes'),
+    createdAt: timestamp('created_at').notNull().defaultNow(),
+    createdBy: uuid('created_by').references(() => users.id),
+    updatedAt: timestamp('updated_at'),
+    updatedBy: uuid('updated_by').references(() => users.id),
+  },
+  (chat_feedback) => ({
+    botIdIdx: index('chat_feedback_id_idx').on(chat_feedback.chatId),
```

Comment:
May need to fix the naming here. 
Could close other PR. Seem this PR already cover last PR.